pkg/assetstorer/postgres.go

```diff
@@ -0,0 +1,8 @@
+package assetstorer
+
+import "github.com/asecurityteam/ipam-facade/pkg/domain"
+
+// PostgresPhysicalAssetStorer stores physical assets in a PostgreSQL database.
+type PostgresPhysicalAssetStorer struct {
```

Comment:
question: where's the *beef?

*beef in this instance refers to the rest of this struct.

Reply:
in a later PR.

Reply:
Sure, but why is it here at all? It doesn't appear to be used anywhere in this PR.

Reply:
Not sure if this is good enough justification, but I'm using this in a branch based of this branch, which I'll open for PR after this is merged.